<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphicsAdvanced</class>
 <widget class="QWidget" name="ConfigureGraphicsAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>Advanced Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="use_accurate_gpu_emulation">
          <property name="text">
           <string>Use accurate GPU emulation (slow)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_vsync">
          <property name="toolTip">
           <string>VSync prevents the screen from tearing, but some graphics cards have lower performance with VSync enabled. Keep it enabled if you don't notice a performance difference.</string>
          </property>
          <property name="text">
           <string>Use VSync (OpenGL only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="force_30fps_mode">
          <property name="text">
           <string>Force 30 FPS mode</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QLabel" name="af_label">
            <property name="text">
             <string>Anisotropic Filtering:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="anisotropic_filtering_combobox">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16x</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
